domains/vyxo.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Design. Develop. Deliver",
+  "repo": "https://github.com/vyxo/vyxo.github.io",
+  "owner": {
+    "username": "vyxo"
```

Comment:
you need an email.

if you dont have email or don't want to share, please leave it `""` and share any of your other contact such as twitter, discord etc